src/release/breaking-changes/wide-gamut-framework.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migration guide for wide gamut Color
+description: Description on Color changes for wide gamut support and migration
+             instructions.
+---
+
+## Summary
+
+The API for the [`Color`][] class in `dart:ui` is changing to support wide gamut
+colors.
+
+## Context
+
+Wide gamut color support was [added to the renderer][] previously. That support
+is now being plumbed up [to the Flutter Framework][] so those colors can be used
+by UIs.
+
+## Description of change
+
+Changes to [`Color`][]:
+
+ 1. Adds a enum field which will specify its [`ColorSpace`][].
+ 1. Adds API to use normalized floating-point color components.
+ 1. Removes API that uses 8-bit unsigned integer color components that can lead
+    to data loss.
+
+Changes to [`ColorSpace`][]:
+
+ 1. `displayP3` will be added.
+
+## Migration guide
+
+### 8-bit unsigned integer constructors
+
+Constructors like `Color.fromARGB` will remain unchanged and have continued
+support. If one wants to take advantage of Display P3 colors they will have to
+use the new `Color.from` constructor that takes normalized floating-point
+color components.
+
+### Implementors of `Color`
```

Comment:
Do we want to add something about (perhaps) `Color` no longer being implementable in the far future?

Reply:
Added a note.